.changeset/large-mugs-fetch.md

```diff
@@ -0,0 +1,6 @@
+---
+"@lens-protocol/api-bindings": minor
+"@lens-protocol/react": minor
```

Comment:
Sorry, one last thing, can you please add `@lens-protocol/react-web` here? We need to update the deps with the updated `@lens-protocol/react`.
```suggestion
"@lens-protocol/react": minor
"@lens-protocol/react-web": minor
```